string callback_service

---

uint8 result
